lib/util/template/index.ts

```diff
@@ -28,6 +28,8 @@ handlebars.registerHelper('containsString', (str, subStr) =>
 
 handlebars.registerHelper('equals', (arg1, arg2) => arg1 === arg2);
 
+handlebars.registerHelper('includes', (arg1: string[], arg2: string) => arg1.includes(arg2));
```

Comment:
```suggestion
handlebars.registerHelper('includes', (arg1: unknown, arg2: unknown) => arg1.includes(arg2));
```

validate args to ensure safety.

@rarkins  log error for invalid args or throw?

Reply:
Ideally throw a config error but I'm not sure how easy that would be so late in the process

Reply:
The throw config error also applies to other helpers, right?

Reply:
Depends whether handlebars swallows them or throws, but potentially yes

Reply:
Ok, baby steps! Let me get this functionality completed, tested and then if we want to do this enhancement here or as part of another PR!

Reply:
validation for the new helper should be done in this PR

Reply:
Ok! You got it :)

Reply:
Ok, so @viceice and @rarkins what do you propose for the error handle of the handlebar?

Reply:
I added some a guard fo the handlebar :)